ext/nokogiri/nokogiri.h

```diff
@@ -23,6 +23,7 @@
 #  define NOKOPUBVAR extern
 #endif
 
+#include <ruby/defines.h> // https://github.com/sparklemotion/nokogiri/issues/2696
```

Comment:
Maybe it should be even above `#ifdef _MSC_VER`? Since there are some includes in that block on Windows.

Reply:
Sure, let's give it a whirl and see what CI says.